config/core.entity_view_display.user.user.default.yml

```diff
@@ -13,16 +15,17 @@ targetEntityType: user
 bundle: user
 mode: default
 content:
-  masquerade:
-    settings: {  }
-    third_party_settings: {  }
-    weight: 1
-    region: content
   member_for:
-    settings: {  }
+    weight: 5
+    region: content
+  user_picture:
+    type: image
+    label: hidden
+    settings:
+      image_link: content
+      image_style: thumbnail
```

Comment:
whoops actually @attiks this seems to have undone the work in HRINFO-1206 to prevent the display or editing of user pictures.

Reply:
Can you fix it (again)?

Reply:
no problem 👍